From 34e189780c29b8ae0d4a726e7509a67049dbac82 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Mon, 18 Nov 2024 15:04:17 +0100
Subject: [PATCH 1374/1695] dts: stm32: rename scmi regulators by regulator@0
 for stm32mp23x

Use a scmi regulator node name coherent with scmi binding:
protocol@17{
	regulators {
		regulator@0 {
		};
		[...]
	};
};

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 860e11b2132281289cd0366b59bd934722fcd818 dts: stm32: rename scmi regulators by regulator@0 for stm32mp23x boards
0afa42ff29fcb6c5e98cb0073a689390e5a34e3f dts: stm32: rename scmi regulators by regulator@0 for stm32mp23x]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: If98ac8d06dedbfa56da93e38ac9ebe9723d3c56d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/419307
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp231.dtsi | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp231.dtsi b/core/arch/arm/dts/stm32mp231.dtsi
index 804f2b93d..0d521d3be 100644
--- a/core/arch/arm/dts/stm32mp231.dtsi
+++ b/core/arch/arm/dts/stm32mp231.dtsi
@@ -1136,31 +1136,31 @@
 			scmi_regulator: protocol@17 {
 				reg = <0x17>;
 
-				scmi_regu: regulators {
+				regulators {
 					#address-cells = <1>;
 					#size-cells = <0>;
 
-					voltd-vddio1 {
+					regulator@0 {
 						reg = <VOLTD_SCMI_VDDIO1>;
 						voltd-supply = <&vddio1>;
 					};
 
-					voltd-vddio2 {
+					regulator@1 {
 						reg = <VOLTD_SCMI_VDDIO2>;
 						voltd-supply = <&vddio2>;
 					};
 
-					voltd-vddio3 {
+					regulator@2 {
 						reg = <VOLTD_SCMI_VDDIO3>;
 						voltd-supply = <&vddio3>;
 					};
 
-					voltd-vddio4 {
+					regulator@3 {
 						reg = <VOLTD_SCMI_VDDIO4>;
 						voltd-supply = <&vddio4>;
 					};
 
-					voltd-vdda18adc {
+					regulator@7 {
 						reg = <VOLTD_SCMI_ADC>;
 						voltd-supply = <&vdda18adc>;
 					};
@@ -2153,7 +2153,7 @@
 					#address-cells = <1>;
 					#size-cells = <0>;
 
-					voltd-vdd33ucpd {
+					regulator@5 {
 						reg = <VOLTD_SCMI_UCPD>;
 						voltd-supply = <&vdd33ucpd>;
 					};
-- 
2.47.2

